app/actions/websocket/index.ts

```diff
@@ -86,6 +87,7 @@ async function doReconnect(serverUrl: string, groupLabel?: BaseRequestGroupLabel
     setTeamLoading(serverUrl, false);
 
     await fetchPostDataIfNeeded(serverUrl, groupLabel);
+    await fetchScheduledPosts(serverUrl, currentTeamId, true, groupLabel);
```

Comment:
If left on entry, this is a duplicate call

Reply:
You mean this one and the entry call would both happen on app start?

Reply:
Yes, see how the connection of the websocket calls entry and then some other stuff

Reply:
Done, removed from here.